.github/workflows/ci.yaml

```diff
@@ -152,6 +152,17 @@ jobs:
           KUBECONFIG=~/.kube/config VERSION=${{ github.sha }} DOCKER_PUSH=true make start
 
       - name: Run tests
+        env:
+          GOPATH: /home/runner/go
+        run: KUBECONFIG=~/.kube/config VERSION=${{ github.sha }} DOCKER_PUSH=true make test-e2e
+
+      - name: Install Numaplane with dataLossPrevention=true
+        env:
+          GOPATH: /home/runner/go
+        run:
+          KUBECONFIG=~/.kube/config VERSION=${{ github.sha }} DOCKER_PUSH=true TEST_MANIFEST_DIR=tests/manifests/special-cases/pause make start
+
+      - name: Run tests with dataLossPrevention=true
```

Comment:
first run the test according to `make start` which has the feature flag off, then with feature flag on